builder/package.ubuntu-1804

```diff
@@ -0,0 +1,49 @@
+if [[ ! -d /tmp/output/ubuntu-1804 ]]; then
+  mkdir -p /tmp/output/ubuntu-1804
+fi
+
+fpm \
+  -s dir \
+  -t deb \
+  -v ${BUILD_NO} \
+  -n R-${R_VERSION} \
+  --deb-priority "optional" \
+  --deb-field 'Bugs: https://github.com/rstudio/r-builds/issues' \
+  --url "http://www.r-project.org/" \
+  --description "GNU R statistical computation and graphics system" \
+  --maintainer "RStudio Inc https://github.com/rstudio/r-builds" \
+  --license "GPL-2" \
+  -p /tmp/output/ubuntu-1804/ \
+  -d zip \
+  -d unzip \
+  -d libpaper-utils \
+  -d xdg-utils \
+  -d libopenblas-base \
+  -d libbz2-1.0 \
+  -d libc6 \
+  -d libcairo2 \
+  -d libcurl4 \
+  -d libglib2.0-0 \
+  -d libgomp1 \
+  -d libicu60 \
+  -d libjpeg8 \
+  -d liblapack3 \
```

Comment:
This liblapack3 can be removed as well

Reply:
Well spotted - can't believe I missed that from the original commit :man_facepalming: 

The idea - at least as far as it's implemented here is that docker-compose picks up the BUILD_NUMBER from Jenkins and passes it through to the container as BUILD_NO. The packaging script picks it up from there.

I've just pushed a new commit for the docker-compose.yml that has the BUILD_NO/BUILD_NUMBER bits in it for all OSes.

Reply:
I've also pushed another commit which removes liblapack3 per your request, @glin.